service/controller.go

```diff
@@ -372,18 +377,17 @@ func (s *service) CreateVolume(
 				return nil, status.Error(codes.InvalidArgument, utils.GetMessageWithRunID(runID, "source snapshot's cluster name '%s' and new volume's cluster name '%s' doesn't match", snapshotSrcClusterName, clusterName))
 			}
 
-			log.Infof("Creating volume from snapshot ID: '%s'", sourceSnapshotID)
+			log.Infof("will create volume from snapshot ID: '%s'", sourceSnapshotID)
 
 			// Get snapshot path
 			if snapshotSourceVolumeIsiPath, err = isiConfig.isiSvc.GetSnapshotSourceVolumeIsiPath(ctx, sourceSnapshotID); err != nil {
 				return nil, status.Error(codes.Internal, err.Error())
 			}
-			log.Infof("Snapshot source volume isiPath is '%s' accessZone '%s'", snapshotSourceVolumeIsiPath, accessZone)
+			log.Infof("snapshot source volume isiPath is '%s' accessZone '%s'", snapshotSourceVolumeIsiPath, accessZone)
 
 			if snapshotIsiPath, err = isiConfig.isiSvc.GetSnapshotIsiPath(ctx, snapshotSourceVolumeIsiPath, sourceSnapshotID, accessZone); err != nil {
 				return nil, status.Error(codes.Internal, err.Error())
 			}
-			log.Debugf("The Isilon directory path of snapshot is= '%s'", snapshotIsiPath)
```

Comment:
good to retain this debugging statement.

Reply:
Reinstated this log line. I hope you see, what one developer views as too much may actually be valuable to another developer :)